packages/react-i18n/src/index.ts

```diff
@@ -3,7 +3,7 @@ export {I18nContext} from './context';
 export {I18n} from './i18n';
 export {useI18n} from './hooks';
 export {withI18n, WithI18nProps} from './decorator';
-export {translate} from './utilities';
+export {memoizedNumberFormatter, translate} from './utilities';
```

Comment:
we want to expose this memoize function so that `react-i18n` consumers can also avoid memory leaks if they need access to a number formatter.